biscuit-auth/src/format/mod.rs

```diff
@@ -388,10 +424,15 @@ impl SerializedBiscuit {
     ) -> Result<Self, error::Token> {
         let keypair = self.proof.keypair()?;
 
+        // The version block is not directly available, so we don’t take it into account here
+        // `append_serialized` is only used for third-party blocks anyway, so maybe we should make `external_signature` mandatory and not bother
```

Comment:
@Geal what do you think about modifying `append_serialized` to require an external key?

Reply:
the external key is available in `ExternalSignature`

Reply:
I’m talking about making the argument mandatory (it’s an option for now), but the function is never called with `None` in the library itself, and i’m not sure why it is exposed publicly